api.go

```diff
@@ -589,6 +575,14 @@ var successStatus = []int{
 // request upon any error up to maxRetries attempts in a binomially
 // delayed manner using a standard back off algorithm.
 func (c Client) executeMethod(ctx context.Context, method string, metadata requestMetadata) (res *http.Response, err error) {
+	if c.IsOffline() {
```

Comment:
if healthcheck is not enabled, this will always return error since default is offline